odxtools/multiplexercase.py

```diff
@@ -71,7 +71,7 @@ def _mux_case_resolve_odxlinks(self, odxlinks: OdxLinkDatabase, *,
 
     def _resolve_snrefs(self, context: SnRefContext) -> None:
         if self.structure_snref:
-            ddds = odxrequire(context.diag_layer).diag_data_dictionary_spec
+            ddds = odxrequire(odxrequire(context.diag_layer).diag_data_dictionary_spec)
```

Comment:
please move drive-by fixes to their own PR, this PR is massive and un-reviwable as is.